notebook/static/notebook/js/codecell.js

```diff
@@ -171,6 +171,8 @@ define([
             if (that.keyboard_manager) {
                 that.keyboard_manager.enable();
             }
+
+            that.code_mirror.setOption('readOnly', that.is_readonly());
```

Comment:
There's MarkdownCell and RawCell, both in `textcell.js`, but TextCell is the parent class, so if setting it there works, that's good.